src/components/CategoryPicker/categoryPickerPropTypes.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import categoryPropTypes from '../categoryPropTypes';
+
+const propTypes = {
+    /** The report ID of the IOU */
+    reportID: PropTypes.string,
```

Comment:
if it is optional would be good to add a default, if not is better to add `isRequired`

Reply:
Done - 9a241c4